components/browser/session/src/main/java/mozilla/components/browser/session/Session.kt

```diff
@@ -197,6 +198,13 @@ class Session(
         }
     }
 
+    /**
+     * The elapsed time of the load.
+     */
+    var elapsedTimeMS: Int by Delegates.observable(0) {
```

Comment:
I wonder if we should pick something like `-1` as a default to indicate that this is not an actual value.

Reply:
This is definitely not a bad idea! I will make that change! Thank you for the suggestion!

Reply:
Resolved in the latest version of the patch!